tensorflow/tensorflow.bzl

```diff
@@ -403,6 +403,57 @@ def tf_gen_op_wrapper_py(name,
       ],)
 
 
+# Given a list of "ops_libs" (a list of files in the core/ops directory
```

Comment:
Thanks, yeah, we should start moving them down and makes sense to start the Java one in the right place :)